ui/Dockerfile

```diff
@@ -46,9 +46,12 @@ ADD 00-mpm.conf.j2 /tmp
 ADD docker-entrypoint.sh /
 RUN rm /etc/httpd/conf.d/welcome.conf /etc/httpd/conf.d/userdir.conf /etc/httpd/conf.d/ssl.conf
 
+RUN update-crypto-policies --set DEFAULT:SHA1
+
 VOLUME /var/log/httpd
 VOLUME /opt/rucio/etc
 
+EXPOSE 80
```

Comment:
Why also expose 80?

Reply:
There is a redirect set up in apache from port 80 to 443. Since this is a browser based app, it is possible that someone will try to hit the http endpoint. We do not want to return a 404 in that case, rather we should redirect them to port 443

https://github.com/rucio/containers/pull/296/files#diff-9afcba5050148bb6d4094d8a2583b949431c2f7fc955a83d79a93d28f0b88f70R84-R92